src/tools/compiletest/src/runtest.rs

```diff
@@ -2710,6 +2712,25 @@ impl<'test> TestCx<'test> {
         fs::remove_dir(path)
     }
 
+    fn run_js_doc_test(&self) {
+        if let Some(nodejs) = &self.config.nodejs {
+            let out_dir = self.output_base_dir();
+
+            self.document(&out_dir);
+
+            let root = self.config.find_rust_src_root().unwrap();
+            let res = self.cmd2procres(
+                Command::new(&nodejs)
+                    .arg(root.join("src/tools/rustdoc-js/tester.js"))
```

Comment:
Shouldn't this be `rustdoc-js-std`?

Either way, I'm confused why we've only migrated one of the suites and not both to compiletest?

Reply:
The `std` one doesn't require to run `rustdoc`, just to run some JS. So it's not really required on their side.